source/scripts/script.js

```diff
@@ -101,17 +112,28 @@ function showSettings() {
     const settings = document.getElementById("settings-container");
     settings.style.visibility = "visible";
     //settings.style.transform = "translate(100%)";
+
     // Get the list of restrictions from local storage
     const getDietaryRestrictions = JSON.parse(localStorage.getItem("dietaryRestrictions"));
+    const getIntolerancesRestrictions = JSON.parse(localStorage.getItem("intolerancesRestrictions"));
     const dietaryContainerElements = document.getElementById('dietary-container').elements;
+    const intolerancesContainerElements = document.getElementById('intolerances-container').elements;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8408391)
```suggestion
    const intolerancesContainerElements = document.getElementById("intolerances-container").elements;
```